Misc/NEWS.d/next/Library/2017-08-12-13-00-21.bpo-5001.tgVI5U.rst

```diff
@@ -0,0 +1,7 @@
+This is a fix of some of the uninformative asserts in `multiprocessing`,
```

Comment:
I don't think three separate NEWS entries and such a detailed description are warranted for what is a small improvement :-)